masterdata/configuration/programs/programs.csv

```diff
@@ -0,0 +1,3 @@
+Uuid,Void/Retire,Name,Description,Program concept,Outcomes concept,_order:1000
```

Comment:
I see only programs are configured, how about workflows and states? https://github.com/mekomsolutions/openmrs-module-initializer/blob/master/readme/prog.md

Reply:
Yes Mohan updated the worflows aswell now.